acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/signer/SignerResponse.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.tests.dsl.signer;
+
+import tech.pegasys.ethsigner.jsonrpc.response.JsonRpcResponse;
+
+import io.netty.handler.codec.http.HttpResponseStatus;
+
+public class SignerResponse<T extends JsonRpcResponse> {
+
+  private final T rpcResponse;
+  private final HttpResponseStatus status;
+
+  public SignerResponse(final T rpcResponse, final HttpResponseStatus status) {
+    this.rpcResponse = rpcResponse;
+    this.status = status;
+  }
+
+  public T rpcResponse() {
```

Comment:
`jsonRpc` perhaps would be a better name, as the class is called Response ...it can be safely inferred that the methods are related to the response

Reply:
done